server/src/main/java/org/spine3/server/entity/AbstractEntity.java

```diff
@@ -157,12 +160,126 @@ private static IllegalStateException noSuchConstructor(String entityClass, Strin
     }
 
     /**
+     * Obtains constructor for the passed aggregate part class.
+     *
+     * <p>The part class must have a constructor with ID and {@code AggregateRoot} parameters.
+     *
+     * @param entityClass the {@code AggregatePart} class
+     * @param idClass     the class of entity identifiers
+     * @param <E>         the entity type
+     * @param <I>         the ID type
+     * @return the {@code AggregatePart} constructor
+     * @throws IllegalStateException if the entity class does not have the required constructor
+     */
+    public static <E extends AggregatePart, I, R extends AggregateRoot<I>> Constructor<E>
```

Comment:
It isn't clear why it is `public' and `static`. I though that should be hidden from the end-users.